python/ray/serve/deployment.py

```diff
@@ -479,7 +419,6 @@ def options(
             new_deployment_config,
             new_replica_config,
             version=version,
```

Comment:
should we remove version too, or does that require more code changes

Reply:
we should, but I'd rather do it in a separate PR at least

Reply:
this already got bigger than I wanted 😓